ext/RecursiveArrayToolsZygoteExt.jl

```diff
@@ -139,19 +139,19 @@ end
     view(A, I...), view_adjoint
 end
 
-function ChainRulesCore.ProjectTo(a::AbstractVectorOfArray)
-    ChainRulesCore.ProjectTo{VectorOfArray}((sz = size(a)))
-end
-
-function (p::ChainRulesCore.ProjectTo{VectorOfArray})(x::Union{
-        AbstractArray, AbstractVectorOfArray})
-    if eltype(x) <: Number
-        arr = reshape(x, p.sz)
-        return VectorOfArray([arr[:, i] for i in 1:p.sz[end]])
-    elseif eltype(x) <: AbstractArray
-        return VectorOfArray(x)
-    end
-end
+# function ChainRulesCore.ProjectTo(a::AbstractVectorOfArray)
```

Comment:
Please don't comment out code like this. You've done it in a few PRs. Just remove the code if it's to be removed.

Reply:
Sure, will do.